HTTPLauncher.py

```diff
@@ -27,11 +29,34 @@ def GET(self, args=None):
 
         return json.dumps(resJson)
 
+class Status:
+    def GET(self, args=None):
+        client = docker.from_env()
+
+        callsEnded = True
+        gwCount = 0
+        for i in range(8):
+            try:
+                gateway = client.containers.get(f"gw{i}")
+                if gateway.status == "running":
+                    gwCount += 1
+                    callsEnded = False
+            except:
+                break
+
+        inCall = 0
+        for p in psutil.process_iter():
+            inCall += (p.name() == "chrome")
+        inCall = round(inCall / 10) # For one gateway being in a call, there are 10 "chrome" processes
```

Comment:
10 "chrome" processes per call => Is it really fixed, deterministic on every platform ? 
Does not look like a really reliable metric...